mava/systems/jax/config.py

```diff
@@ -138,6 +137,9 @@ def update(self, **kwargs: Any) -> None:
                         system. Perhaps try adding the component using .add() \
                         in the system builder."
                     )
+            elif isinstance(dataclass, SimpleNamespace):
```

Comment:
This would also need to be added in the add() method a few line above

Reply:
Great catch! I updated the code.